modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -3,193 +3,114 @@
 [[the-test-directory-structure]]
 == The Test Directory Structure
 
-This is the structure of acceptance directory inside 
+This is the structure of the acceptance directory inside
 link:https://github.com/owncloud/core[the core repository's] `tests` directory:
```

Comment:
link: in front of http is not necessary.
http is its own prefix (link: is only needed in special cases)

Reply:
That was existing crud, I fixed it.